internal/descriptor/descriptor.go

```diff
@@ -46,3 +46,8 @@ func GetTitleOrMediaType(desc ocispec.Descriptor) (name string, isTitle bool) {
 	}
 	return name, true
 }
+
+// GenerateContentKey generates a unique key for each content descriptor using digest and name.
```

Comment:
NIT: better keep comment length less than 80

```suggestion
// GenerateContentKey generates a unique key for each content descriptor using
// digest and name.
```